client/asset/btc/btc.go

```diff
@@ -5792,3 +5794,74 @@ func deserializeMsgTx(r io.Reader) (*wire.MsgTx, error) {
 func msgTxFromBytes(txB []byte) (*wire.MsgTx, error) {
 	return deserializeMsgTx(bytes.NewReader(txB))
 }
+
+func (btc *baseWallet) ConfirmRedemption(coinID dex.Bytes, redemption *asset.Redemption, feeSuggestion uint64) (*asset.ConfirmRedemptionStatus, error) {
+	txHash, _, err := decodeCoinID(coinID)
+	if err != nil {
+		return nil, err
+	}
+
+	_, confs, err := btc.rawWalletTx(txHash)
```

Comment:
> to see if the fee needs to be bumped

Maybe could treat the redeem tx, which has a single output, as a transaction comprising a single change output.

I'm not excited about having Core start tracking a new set of "acceleration coins" for the redeem though.

We can make a TODO here unless it's simpler than I think.

However, if it doesn't get purged from mempool, I think it's alright.  The user just has to wait longer.  Still, there's a subtle possible vuln if you are the maker because your redeem really shouldn't hang around unmined after the taker's contract has expired.  But we've survived without this so far.

Reply:
I don't think neutrino has a purge transaction option like dcrwallet spv does either. So even if you thought a tx needed to go I don't think there's anything you can do about it?

Reply:
So a kvdb for dcr to track a chain of accelerations of the redeem.  I guess so.  Feels messy and error prone.

Reply:
Does the accelerator drop the old tx or does the wallet handle that? Need to look.

Reply:
It's doesn't drop or replace.  It spends the change. CPFP.
This is what I was getting at a few comments above.

Reply:
I was proposing to do RBF. I think the server might have to be updated for this to start searching for the replacement tx instead of the original one though.

> So a kvdb for dcr to track a chain of accelerations of the redeem. I guess so. Feels messy and error prone.

I agree, it's kind of annoying in ETH, but probably worth it since BTC can have fee spikes. Since it also could require server changes I think it can be done later as a separate piece of work.

Reply:
ETH is doing RBF/replace nonce, not CPFP. Couldn't we do the same thing here?

Reply:
RBF is not a thing in Decred.  And in BTC it's not trivial (need to fiddle with sequence values) or without consequences (new txid and any spenders of the replaced 0-conf UTXOs are hosed so we have to prevent that like our lives depend on it).

Reply:
`CPFP` oh yeah it ups the entire fee of the chain of txn. I remember now.